### PR TITLE
WIP: Add readiness probe

### PR DESCRIPTION
**Pull request by fkautz**

### CODE REVIEW

File: controlplane/cmd/nsmd/nsmd.go [hunk 1]
```diff
@@ -17,14 +18,18 @@ func main() {
 	model := model.NewModel(nsmUrl)
 	defer nsmd.StopRegistryClient()
 
+	// spin up registry client
+	_, err := nsmd.RegistryClient()
+	if err != nil {
+		logrus.Fatalf("Error starting registry client")
```
Comment (nickolaev) [inline controlplane/cmd/nsmd/nsmd.go]: Fatalln

File: controlplane/cmd/nsmd/nsmd.go [hunk 2]
```diff
@@ -35,5 +40,9 @@ func main() {
 		<-c
 		wg.Done()
 	}()
+	err = ioutil.WriteFile("/tmp/online", []byte("online"), 555)
+	if err != nil {
+		logrus.Fatalf("Error writing /tmp/online readiness check", err)
```
Comment (nickolaev) [inline controlplane/cmd/nsmd/nsmd.go]: Add %v in the format string

File: examples/cmd/nsc/nsc.go [hunk 3]
```diff
@@ -76,7 +76,7 @@ func main() {
 	reply, err := nsmConnectionClient.Request(context.Background(), request)
 
 	if err != nil {
-		logrus.Errorf("failure to request connection with error: %+v", err)
+		logrus.Fatalln("failure to request connection with error: %+v", err)
```
Comment (nickolaev) [inline examples/cmd/nsc/nsc.go]: Fatalf

File: k8s/conf/nsmd.yaml [hunk 4]
```diff
@@ -11,21 +11,28 @@ spec:
     spec:
       containers:
         - name: nsmdp
-          image: networkservicemesh/nsmdp
+          image: networkservicemesh/nsmdp:5645df33
```
Comment (nickolaev) [inline k8s/conf/nsmd.yaml]: Why is that?

File: controlplane/cmd/nsmd/nsmd.go [hunk 5]
```diff
@@ -35,5 +40,9 @@ func main() {
 		<-c
 		wg.Done()
 	}()
+	err = ioutil.WriteFile("/tmp/online", []byte("online"), 555)
```
Comment (nickolaev) [inline controlplane/cmd/nsmd/nsmd.go]: Should it be 0555?

File: k8s/conf/nsmd.yaml (hunk 4 above)
Comment (fkautz) [reply k8s/conf/nsmd.yaml]: you caught me experimenting :p

**Pull request closed without merging**